Gemfile

```diff
@@ -14,6 +14,8 @@ group :development, :unit_tests do
   gem 'metadata-json-lint',                               :require => false
   gem 'puppet-blacksmith',                                :require => false
   gem 'rest-client', "1.6.8",                             :require => false
+  gem 'syck',                    :require => false
```

Comment:
done, though I would still need to make the requested change to constrain this to certain versions of ruby, which I believe I had no success with last time I looked at this patch.  Happy to make changes if there are suggestions how though.